linera-chain/src/data_types.rs

```diff
@@ -82,6 +82,19 @@ impl Block {
         blob_ids
     }
 
+    /// Returns all the read blob IDs in this block's operations.
+    #[cfg(with_testing)]
+    pub fn read_blob_ids(&self) -> HashSet<BlobId> {
```

Comment:
Is this used anywhere?

Reply:
It was at some point ha but not anymore. Will remove